src/frontend/src/components/public/PIAFormTabs/review/viewReviewSection.tsx

```diff
@@ -51,20 +51,22 @@ const ViewReviewSection = (props: IReviewProps) => {
         <div className="mt-2">{reviewSection?.reviewedByDisplayName}</div>
       </div>
 
-      {reviewGuid === getGUID() && !printPreview && (
-        <div className=" col d-flex justify-content-end">
-          <button
-            className="bcgovbtn bcgovbtn__tertiary p-3"
-            onClick={() => {
-              if (editReviewNote) editReviewNote(true);
-              else if (stateChangeHandler) setEditReview(true);
-            }}
-          >
-            <FontAwesomeIcon className="ms-1" icon={faFileEdit} size="lg" />
-            Edit review
-          </button>
-        </div>
-      )}
+      {reviewGuid === getGUID() &&
+        !printPreview &&
+        pia.status !== PiaStatuses.COMPLETE && (
```

Comment:
I don't see how this can be in the state table. The reviewGuid is checked at runtime and so is printPreview, based on action. Any thoughts?

Reply:
so, I would make this check into a function:
editReviewNote() {
if (reviewGuid === getGUID() &&
        !printPreview &&
statusList[pia.status][pages]['review][param]['editReviewNote']
) {
return true;
}
return false;
}

and in the state table under Pages - >Review add a parameter called 'params' that can be page specific and create a param called editReviewNote

for most status it will be true except for completed.